### PR TITLE
make_report example segfault

### PR DESCRIPTION
**Pull request by massich**
related to #5893.

This is what I get on my machine: 

``` shell
~/code/mne-python report_segfault*
(mne) ❯ ipython examples/visualization/make_report.py         
Fatal Python error: Segmentation fault

Thread 0x00007f6211648700 (most recent call first):
  File "/home/sik/miniconda3/envs/mne/lib/python3.6/threading.py", line 295 in wait
  File "/home/sik/miniconda3/envs/mne/lib/python3.6/threading.py", line 551 in wait
  File "/home/sik/miniconda3/envs/mne/lib/python3.6/site-packages/IPython/core/history.py", line 829 in run
  File "/home/sik/miniconda3/envs/mne/lib/python3.6/site-packages/IPython/core/history.py", line 58 in needs_sqlite
  File "<decorator-gen-24>", line 2 in run
  File "/home/sik/miniconda3/envs/mne/lib/python3.6/threading.py", line 916 in _bootstrap_inner
  File "/home/sik/miniconda3/envs/mne/lib/python3.6/threading.py", line 884 in _bootstrap

Current thread 0x00007f621490a740 (most recent call first):
  File "/home/sik/miniconda3/envs/mne/lib/python3.6/site-packages/tvtk/tvtk_base.py", line 595 in _wrap_call
  File "tvtk_classes/open_gl_render_window.py", line 79 in get_pixel_data
  File "/home/sik/miniconda3/envs/mne/lib/python3.6/site-packages/mayavi/tools/figure.py", line 348 in screenshot
  File "/home/sik/code/mne-python/mne/report.py", line 150 in _iterate_trans_views
  File "/home/sik/code/mne-python/mne/report.py", line 1965 in _render_trans
  File "/home/sik/code/mne-python/mne/report.py", line 298 in _iterate_files
  File "/home/sik/code/mne-python/mne/report.py", line 1470 in <genexpr>
  File "/home/sik/code/mne-python/mne/report.py", line 1470 in parse_folder
  File "/home/sik/code/mne-python/mne/utils/_logging.py", line 74 in verbose
  File "<string>", line 2 in parse_folder
  File "/home/sik/code/mne-python/examples/visualization/make_report.py", line 30 in <module>
  File "/home/sik/miniconda3/envs/mne/lib/python3.6/site-packages/IPython/utils/py3compat.py", line 188 in execfile
  File "/home/sik/miniconda3/envs/mne/lib/python3.6/site-packages/IPython/core/interactiveshell.py", line 2683 in safe_execfile
  File "/home/sik/miniconda3/envs/mne/lib/python3.6/site-packages/IPython/core/shellapp.py", line 323 in _exec_file
  File "/home/sik/miniconda3/envs/mne/lib/python3.6/site-packages/IPython/core/shellapp.py", line 391 in _run_cmd_line_code
  File "/home/sik/miniconda3/envs/mne/lib/python3.6/site-packages/IPython/core/shellapp.py", line 271 in init_code
  File "/home/sik/miniconda3/envs/mne/lib/python3.6/site-packages/IPython/terminal/ipapp.py", line 323 in initialize
  File "/home/sik/miniconda3/envs/mne/lib/python3.6/site-packages/traitlets/config/application.py", line 87 in catch_config_error
  File "<decorator-gen-112>", line 2 in initialize
  File "/home/sik/miniconda3/envs/mne/lib/python3.6/site-packages/traitlets/config/application.py", line 657 in launch_instance
  File "/home/sik/miniconda3/envs/mne/lib/python3.6/site-packages/IPython/__init__.py", line 125 in start_ipython
  File "/home/sik/miniconda3/envs/mne/bin/ipython", line 11 in <module>
zsh: segmentation fault (core dumped)  ipython examples/visualization/make_report.py

```

let's see what travis has to say

### CODE REVIEW

**Comment by sappelhoff** [comment]
slightly different but same result here:

```shell
(mne) stefanappelhoff@arc-lin-004309:~/Desktop/bids/mne-python$ ipython examples/visualization/make_report.py 
Embedding : jquery-1.10.2.min.js
Embedding : jquery-ui.min.js
Embedding : bootstrap.min.js
Embedding : jquery-ui.min.css
Embedding : bootstrap.min.css
Iterating over 28 potential files (this may take some time)
Rendering : /home/stefanappelhoff/mne_data/MNE-sample-data/MEG/sample/all-trans.fif
Using outer_skin.surf for head surface.
Getting helmet for system 306m
Segmentation fault (core dumped)

```

File: mne/report.py [hunk 1]
```diff
@@ -154,6 +158,10 @@ def _iterate_trans_views(function, **kwargs):
     for view, ax in zip(views, axes):
         mlab.view(view[0], view[1])
         if fig.scene is not None:
+            try:
+                mlab.process_ui_events()
```
Comment (jasmainak) [inline mne/report.py]: not sure if this helps or if it's the same. I have this in one of my scripts:

```py
from pyface.api import GUI
GUI().process_events()
```

copied from some github issue ...

**Comment by larsoner** [comment]
There was a commit one month ago to Mayavi that might fix it:

https://github.com/enthought/mayavi/pull/771

Can you see if updating Mayavi fixes the problem?

**Comment by larsoner** [comment]
I think I found a fix, PR coming hopefully soon

**Pull request closed without merging**